### ISSUE TITLE
Feature request: add possibility to reposition the relationship's label

### ISSUE THREAD

**Issue by gparmeggiani** [likec4/likec4#1210]
Sometimes the label is places in a non ideal spot, too close to other relationships. 
Being able to reposition the label to a different place would allow for more clarity in the diagram

It would be great if you could add this possibility in the visual editor

**Comment by xeor** [comment]
Have you tried the auto-layout functionality? Maybe it helps.. You can also drag move things around a little and it generates a comment with the alignment above the element.

**Comment by gparmeggiani** [comment]
yes, I'm referring to this exact feature, however my understanding is that you can drag move the "boxes" and the relationship edges, but no way to reposition the edges "titles". Is this already available?
